mappings/net/minecraft/world/SectionDistanceLevelPropagator.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_4079 net/minecraft/util/SectionDistanceLevelPropagator
+CLASS net/minecraft/class_4079 net/minecraft/world/SectionDistanceLevelPropagator
```

Comment:
:+1: finally seeing something that makes some sense

Reply:
Closes #947